internal/services/databricks/validate/databricks_virtual_network_peering_id_test.go

```diff
@@ -0,0 +1,88 @@
+package validate
```

Comment:
this file shouldn't be needed anymore, since it's available within `go-azure-sdk`?

Reply:
Fixed.